core/site_scons/tools.py

```diff
@@ -157,21 +157,26 @@ def cargo_build():
             "-Z build-std-features=panic_immediate_abort",
         ]
 
+        rustc_flags = []
```

Comment:
Very nit: since `-Z` options are for unstable/nightly options only could you perhaps reflect that in the variable name? Might save some minutes when eventually someone tries to make this compile on stable release:sweat_smile:

Reply:
Sounds good, fixed in 41ee00db2659dedc339c8644344e2e6d576ff006.